modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="creating_machine_pools_ocm_{context}"]
+ifndef::openshift-rosa[]
+= Creating a machine pool
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+= Creating a machine pool using OpenShift Cluster Manager
+endif::openshift-rosa[]
+
+ifndef::openshift-rosa[]
+A default machine pool is created when you install an {product-title} cluster. After installation, you can create additional machine pools for your cluster by using {cluster-manager}.
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+You can create additional machine pools for your {product-title} (ROSA) cluster by using {cluster-manager}.
+endif::openshift-rosa[]
+
+[IMPORTANT]
+====
+The compute node instance types, autoscaling options, and node counts that are available to you depend on your 
+ifdef::openshift-rosa[]
+ROSA 
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+{product-title} 
+endif::[]
+subscriptions, resource quotas and deployment scenario. For more information, contact your sales representative or Red Hat support.
+====
+
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You created a ROSA cluster.
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+* You created an {product-title} cluster.
+endif::[]
+
+.Procedure
+
+. From {cluster-manager-url}, navigate to the *Clusters* page and select your cluster.
+
+. Under the *Machine pools* tab, click *Add machine pool*.
+
+. Add a *Machine pool name*.
+
+. Select a *Worker node instance type* from the drop-down menu. The instance type defines the vCPU and memory allocation for each compute node in the machine pool.
+
+. Optional: Configure autoscaling for the machine pool:
+.. Select *Enable autoscaling* to automatically scale the number of machines in your machine pool to meet the deployment needs.
+.. Set the minimum and maximum node count limits for autoscaling. The cluster autoscaler does not reduce or increase the machine pool node count beyond the limits that you specify.
+** If you deployed your cluster using a single availability zone, set the *Minimum and maximum node count*. This defines the minimum and maximum compute node limits in the availability zone.
+** If you deployed your cluster using multiple availability zones, set the *Minimum nodes per zone* and *Maximum nodes per zone*. This defines the minimum and maximum compute node limits per zone.
++
+[NOTE]
+====
+Alternatively, you can set your autoscaling preferences for the machine pool after the machine pool is created.
+====
+
+. If you did not enable autoscaling, select a compute node count:
```

Comment:
Thanks for this suggestion. I would prefer to use "compute node" instead of "worker node" where possible, except where referring to UI and CLI objects. I will discuss this within the docs team and consider how to best make this consistent in the assembly.

Reply:
@0kashi I have now resolved this throughout the assembly, based on our conversation on the call a couple of days ago.